### PR TITLE
Use custom LoggingFilter for jersey

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos @hmunfru

### CODE REVIEW

**Comment by jframos** [comment]
LGTM

File: rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/util/MyLoggingFilter.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+/**
+ * Extend LoggingFilter in order to disable stderr traces for jersey
+ */
+public class MyLoggingFilter extends LoggingFilter {
+
+    private static Logger log = LoggerFactory.getLogger(MyLoggingFilter.class);
+
+    public MyLoggingFilter() {
+        super(null, true);
+        System.out.println("MyLoggingFilter running");
```
Comment (flopezag) [inline rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/util/MyLoggingFilter.java]: Why system.out.println?

**Comment by hmunfru** [comment]
LGTM

File: rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/util/MyLoggingFilter.java (hunk 1 above)
Comment (flopezag) [inline rest-api/src/main/java/com/telefonica/euro_iaas/paasmanager/rest/util/MyLoggingFilter.java]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**